hw/ip/tlul/tlul.core

```diff
@@ -12,6 +12,7 @@ filesets:
       - lowrisc:tlul:socket_m1
       - lowrisc:tlul:adapter_sram
       - lowrisc:tlul:adapter_reg
+      - lowrisc:tlul:adapter_host
```

Comment:
The goal for this file is to eventually go away; please don't add new functionality to it. Other cores should depend on `lowrisc:tlul:adapter_host` directly.